RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress+View.swift

```diff
@@ -26,18 +26,18 @@ public extension AccountRecoveryScanInProgress {
 		let active: IdentifiedArrayOf<Profile.Network.Account>
 		let hasFoundAnyAccounts: Bool
 		let maxIndex: Int
+
+		var buttonControlState: ControlState {
+			isScanInProgress ? .disabled : .enabled
+		}
+
 		var isScanInProgress: Bool {
 			switch status {
 			case .scanComplete: false
 			default: true
```

Comment:
Since there is a default case, you might as well just put `var isScanInProgress: Bool { status != .scanComplete }`, unless it's non-equatable?

Reply:
sure! was playing around with different impl earlier, but now we are reaching feature impl stability